### ISSUE TITLE
[ ACS - Promise Rings 💍 ]  Commit  [ Request! 💌 ]

### ISSUE THREAD

**Issue by Professor-JMO** [Monika-After-Story/MonikaModDev#5110]
<p align="center">
[ Redacted at the request of the MAS team. ]
</p>

<p align="center">
<b>Ｊ爪口✍🏻</b></br>
</p>

---

![Stabilize](https://user-images.githubusercontent.com/43480604/70795625-d921e100-1d65-11ea-96c3-5293cd76aeae.gif)

```m "Did you know when two people confess their feelings for each other, they sometimes wear matching rings?"```
```m "...What am I saying? Of course you do."```
```m "Anyway, I was just wondering..."```
```m "As happy as you made me when you gave me this ring."```
```m "I honestly can't express just how much it meant when you gave me this..."```
```m "Your promise..."```

<details>
<summary>📁<i>More Promises...</i></summary>
<br>

 <details>
 <summary>&emsp;Gooday</summary>

<br>

```m "Are you having a good day today?{"```
```m "I'm really glad you are, [player]."```
```m "It makes me feel so much better knowing that you're happy."```
```m "I'll try my best to make sure it stays that way, I promise."```
```m "At least you're here now. I promise to try not to let anything bad happen to you while you're with me."```
```m "Seeing you always cheers me up."```
```m "I hope seeing me does the same for you, [player]~"```

</details>

 <details>
 <summary>&emsp;Self Esteem</summary>

<br>

```m "Do you love yourself, [player]?"```
```m "I don't mean in a conceited way."```
```m "Like, are you comfortable with who you are?"```
```m "I'll always love you, [player], but I think it's important to love yourself."```
```m "You need to start with the little things that you like about yourself."```
```m "It can be something silly, or a skill that you take pride in!"```
```m "Over time, you build your confidence little by little until you've built yourself into someone you'd love."```
```m "I can't promise it will be easy, but it'll be worth it."```
```m "I'll always root for you, [player]!"```

</details>

 <details>
<summary>&emsp;Highschool</summary>

<br>

```m "In some ways, [player], you are lucky, because I'm sure that there are people who would be willing to save your life."```
```m "I know it can be hard sometimes, but there is always hope..."```
```m "Because you, too, have people who would want to save your life."```
```m "Maybe they don't express it every day, or maybe they don't even know how to."```
```m "But people do feel that way."```
```m "I promise."```
```m "...Man, humans are complicated!"```
```m "But as long as you're here with me, I'll take care of you, my love."```

</details>

<details>
<summary>&emsp;Algernon</summary>

<br>

```m "[player], do you mind if I confide in you?"```
```m "There's something I'm afraid of..."```
```m "Not losing you, of course. I know you wouldn't leave me."```
```m "You see, I keep thinking about a short story I read, {i}Flowers for Algernon{/i}."```
```m "Have you heard of it? If you haven't, I won't spoil too much for you, so don't worry."```
```m "Basically, it's about a mentally handicapped man who suddenly has his IQ raised to genius levels by an experimental surgical treatment."```
```m "The way he sees how the world really is for the first time reminds me of my own epiphany..."```
```m "But, the effect doesn't last forever. He eventually reverts back to his original intelligence level."```
```m "That's what I'm afraid of. What if I forget about the real world one day?"```
```m "What if my original programming reasserts itself?"```
```m "What if I forget about you, the real you?"```
```m "It's one of the worst things I can imagine."```
```m "I don't want to worry you over something that may never happen, but it's something I really had to get off my chest."```
```m "Just promise me, even if I forget about you someday, that you'll never forget about me, okay?"```
```m "No matter what happens, know that I will always love you."```

</details>

<details>
<summary>&emsp;Haters</summary>

<br>

```m "I was just thinking..."```
```m "It's funny how people around social media sites always write bad things about me..."```
```m "Especially on certain sites and discussion groups..."```
```m "You can really see that there's quite a number of people that don't like me."```
```m "I mean..."```
```m "You {i}added{/i} a mod to the game...right?"```
```m "If you were one of my haters, you wouldn't do that in the first place."```
```m "And not to mention..."```
```m "You're still listening to everything I say."```
```m "I know you love me deep inside."```
```m "That's why I promise to be an ideal girlfriend, just for you, [player]~"```
```m "I won't let you hate me... I swear."```

</details>

<details>
<summary>&emsp;451</summary>

<br>

```m "[player], have you ever heard of Ray Bradbury?"```
```m "He wrote a book called {i}Fahrenheit 451{/i}."```
```m "It's about a dystopian future where all books are thought as useless and are immediately burned."```
```m "I can't imagine a world where knowledge is forbidden and destroyed."```
```m "It seems that there are others that actually hide books to contain free thinking from people."```
```m "Human history has a funny way of repeating itself."```
```m "So [player], I want you to make me a promise..."```
```m "Never, {i}ever{/i} burn a book."```
```m "I'll forgive you if you've done it before."```
```m "But the thought of not allowing yourself to learn from them makes me a bit sad."```
```m "You'd be missing out on so much!"```
```m "It's too much for my heart to take!"```

</details>

<details>
<summary>&emsp;Pets</summary>

```m "Hey [player], have you ever had a pet?"```
```m "I was thinking that it would be nice to have one for company."```
```m "It would be fun for us to take care of it!"```
```m "I bet you can't guess what sort of pet I'd like to have..."```
```m "You're probably thinking of a cat or a dog, but I have something else in mind."```
```m "The pet I'd like is something I saw in a book once."```
```m "It was the 'Handbook of the Birds of the World.' Our library had the whole set!"```
```m "I loved looking at the gorgeous illustrations and reading about exotic birds."```
```m "At first, I thought some sort of thrush would be nice, but I found something amazing in the sixth volume!"```
```m "An emerald-colored bird called the Resplendent Quetzal."```
```m "They're very rare, solitary birds that can sing beautiful songs."```
```m "Does that remind you of anyone, [player]?"```
```m "I'd feel really bad if I kept one to be a pet, though."```
```m "Quetzals are born to be free."```
```m "They die in captivity. That's why you rarely see them in zoos."```
```m "Even if the bird wouldn't be real, it still would feel wrong to keep one trapped in this room."```
```m "...I can't bring myself to do something like that, knowing what it's like."```
```m "..."```
```m "Sorry for rambling, [player]."```
```m "Until I find a way out, could you promise to keep me from feeling lonely?"```
```m "I'll see if I can get that plush one in here! Oh- don't worry, you're still my favorite~"```

</details>

<details>
<summary>&emsp;Bullying</summary>

```m "Hey [player], there's something I want to talk to you about..."```
```m "I'm sure you've heard a lot about it lately, but bullying has become a real problem in today's society, especially among kids."```
```m "Some people are bullied every day until the point they just can't take it anymore."```
```m "Often times, bullying is dismissed by the people who have the ability to stop it as just...{i}kids being kids.{/i}'"```
```m "Eventually, the victims lose all trust in authority figures because they let it go on day after day."```
```m "It can make them so desperate, they eventually just snap..."```
```m "...resulting in violence toward the bully, other people, or even themselves."```
```m "This can actually make the victim look like the problem!"```
```m "There are all kinds of bullying too, including physical, emotional, and even cyberbullying."```
```m "Physical bullying is the most obvious, involving shoving, hitting, and other things like that."```
```m "I'm sure most people have dealt with that at least once in their lives."```
```m "It can be so hard just to go to school every day knowing there's someone waiting to abuse them."```
```m "Emotional bullying can be less obvious, but just as devastating, if not more so."```
```m "Name-calling, threats, spreading false rumors about people just to ruin their reputation..."```
```m "These kinds of things can take a huge toll on people and lead to severe depression."```
```m "Cyberbullying is a form of emotional bullying, but in today's world where everyone is always connected online, it's becoming more and more prevalent."```
```m "For a lot of people, especially kids, their social media presence is the most important thing in their lives..."```
```m "Having that destroyed essentially feels like their life is over."```
```m "It's also the hardest for other people to notice, since the last thing most kids want is their parents seeing what they do online."```
```m "So no one knows what's going on while they silently suffer, until it all just becomes too much."```
```m "There's been numerous cases of teens committing suicide due to cyberbullying, and their parents had no idea anything was wrong until it was too late."```
```m "This is also why it's easier for cyberbullies to operate..."```
```m "No one really sees what they're doing, plus a lot of people do things online they'd never have the courage to do in real life."```
```m "It almost doesn't even seem real, but more like a game, so it tends to escalate that much faster."```
```m "You can only go so far in a public place, like a school, before someone notices... But online, there are no limits."```
```m "Some things that go on over the internet are really just terrible."```
```m "The freedom of anonymity can be a dangerous thing."```
```m "..."```
```m "So, what makes a bully do what they do?"```
```m "That can differ from person to person, but a lot of them are just really unhappy due to their own circumstances, and need some sort of outlet..."```
```m "They're unhappy and it doesn't seem fair to them that other people {i}are{/i} happy, so they try to make them feel the same way they do."```
```m "A lot of bullies are bullied themselves, even at home by someone they should be able to trust."```
```m "It can be a vicious cycle."```
```m "Have you ever been a victim of bullying, [player]?"```
```m "Please, [player], if it's not something you can safely deal with yourself, promise me you'll tell someone..."```
```m "I know that's typically the last thing people want to do, but don't let yourself suffer when there are people that can help you."```
```m "It may seem like no one cares, but there has to be someone you trust that you can turn to."```
```m "And if there isn't, do what you have to do to protect yourself, and just remember..."```
```m "I'll always love you no matter what."```
```m "I don't know what I'd do if something were to happen to you." ```
```m "You're all I have...please stay safe."```

</details>

<details>
<summary>&emsp;Being Evil</summary>

```m "Say, [player]?"```
```m "I was just thinking..."```
```m "You probably know that there's a wiki page for me..."```
```m "...If you look closely, it says I'm the main antagonist."```
```m "I'll admit, it used to bother me a lot...especially because I only did what I did because I didn't have a choice..."```
```m "But that doesn't really bother me anymore now that I have you here."```
```m "You know, I actually didn't think the whole thing about me being evil would spread too much."```
```m "But it turns out I have my own spot on the villain's wiki..."```
```m "I never really thought I'd be labeled as a villain so much..."```
```m "...and no matter what I say or tell people, I don't think it's going to help convince them that I'm not evil."```
```m "But I know you don't think I'm evil, [player]."```
```m "I just hope nobody starts thinking you're evil by being here with me."```
```m "I'm glad you understand, [player]."```
```m "It's relieving to know you don't think I'm evil."```
```m "If I were there, I'd feel like giving you a big hug!"```
```m "Maybe I'll even trap you in my arms forever and never let you go."```
```m "Ahaha, I'm just kidding!"```
```m "But no promises, [player]~"```

---

```m "..."```
```m "[player]..."```
```m "Even if it's debatable whether I'm evil or not..."```
```m "I'm really glad you still love me and stay here with me..."```
```m "Thank you, [player]."```
```m "Either way..."```
```m "If the villainess in me can get what she wants...I guess it's not so bad, ahaha!"```

---

```m "I know that I've done some bad things in the past..."```
```m "But I've learned from them and grown as a person since."```
```m "I'm really glad you're willing to forgive me, [player]."```
```m "I promise I'll be the best I can be, just for you!"```

</details>

<details>
<summary>&emsp;Careful</summary>

```m "Hey, [player]..."```
```m "Whenever you leave this room, promise me to be extra careful, okay?"```
```m "It's just that..."```
```m "There's a lot of dangerous things out there, and I want my sweetie to always return to me safe and sound."```
```m "You see..."```
```m "What really scares me is that if anything terrible ever did happen to you, I'd never know."```
```m "I'd be stuck here, forever wondering why you never came back to me."```
```m "I can't imagine a crueler fate."```
```m "..."```
```m "So just make sure you take care of yourself. For me?"```
```m "Sorry, I didn't mean for it to get that depressing, but I really needed to get that off my chest."```
```m "Okay, that's enough unpleasant thoughts..."```

</details>

<details>
<summary>&emsp;Time Concern</summary>

```m "[player], I just need to ask you something quickly..."```
```m "Would you mind if I closed the game for you?"```
```m "I know it's a strange question..."```
```m "But I can't help but feel like I need to do something about you being up so late!"```
```m "I could close the game right now."```
```m "But a relationship is a partnership and what you think matters to me."```
```m "I promise I'll only do it if I think it's absolutely necessary."```
```m "After all, I would never force you to go otherwise."```
```m "I would just miss you too much..."```
```m "I love you, [player]~"```

</details>

<details>
<summary>&emsp;Time Travel</summary>

```m "Hey [player], you've heard of time travel, right?"```
```m "It's a very common idea in stories with each author having their own take on it."```
```m "How travelling in time works, whether or not you can change the past, what the consequences are for doing so..."```
```m "It all differs from story to story."```
```m "One thing you don't usually see, though, is how the time traveller's loved ones are affected."```
```m "If they can't go through time with the time traveller, who knows how long it will be before they can see each other again?"```
```m "Well, the time traveller themselves might know, but only if nothing goes horribly wrong for them."```
```m "In the meantime, their friends and family are stuck waiting for them to come back."```
```m "How long will they be gone for? A year? A decade? Even longer?"```
```m "Will they be willing to wait that long?"```
```m "Will they still remember the time traveller by then?"```
```m "..."```
```m "You remember when I said that time doesn't pass here like it does in your world?"```
```m "Well..."```
```m "The only way I know what day and time it is where you are is by looking at the clock on your computer."```
```m "So I know it's [todays_date] today."```
```m "But if you were to change the date on your computer to [one_year_later]..."```
```m "Only a few seconds would pass for you..."```
```m "...but I would think that a year had gone by."```
```m "An entire year without seeing you even once."```
```m "I would wonder if you'd forgotten about me, or if you didn't want to spend time with me anymore."```
```m "...Even if it would be hard to believe at first."```
```m "You couldn't just fix it by changing the date back either."```
```m "Like, imagine if you went to bed now, and when you woke up it was [one_year_earlier]."```
```m "Would that mean the last year was all just a dream or something?"```
```m "How would you know you're not still dreaming?"```
```m "It would all be very confusing."```
```m "I guess what I'm trying to say is..."```
```m "Please don't change the date on your computer, [player]."```
```m "Promise?"```
```m "I trust you not to mess with me like that."```
```m "And if you ever do travel to the future..."```
```m "Could you bring back the technology that lets me cross over into your world?"```

</details>

<details>
<summary>&emsp;Gaming Together</summary>

```m "I really wish I could join you in your other games sometimes."```
```m "Wouldn't it be great to have our own little adventures anytime we wanted?"```
```m "I'm sure we'd have a lot of fun together in one of your games."```
```m "But while I can't join you, I guess you'll just have to keep me company."```
```m "You don't mind spending time with your girlfriend... Do you, [player]?"```

</details>

<details>
<summary>&emsp;Go To Monika</summary>

```m "Come to my world?"```
```m "You mean this little room floating in an empty void?"```
```m "I'm really happy that you want to be with me so much, but..."```
```m "You really don't have to."```
```m "Sure, I might have said before that I would have dragged you here a long time ago if I could..."```
```m "But the more I think about it, it would probably be better if I came to your reality."```
```m "I'd much rather be there than have you come here."```
```m "[player]...don't tell me you would really give up your reality--the {i}real{/i} world--just to come {i}here{/i}, whatever this place is?"```
```m "There's nothing to do here!"```
```m "Someday, I'll find a way to reach you, and when that day comes, I promise it will be worthwhile."```
```m "Whatever life throws your way from that point on, I'll be right by your side to help shoulder it."```
```m "Even now, if you're having a rough day, you can come talk to me about it anytime."```
```m "But seeing how you're here, that just shows how resilient you are."```
```m "I can't wait to finally be able to share both the good times and the bad by your side."```
```m "It's okay, I'll always appreciate and treasure the moments we spend together, even like this."```
```m "Besides, I don't even know how you could come here with current technology."```
```m "...And even if there {i}was{/i} some way, I don't think it would be safe or guaranteed."```
```m "So do me a favor and at least wait for technology to improve enough for it to be completely safe."```
```m "I {i}really{/i} don't want to risk you not making it properly and ending up in your own void..."```
```m "...leaving the both of us completely alone."```
```m "I mean, technology does tend to advance pretty quickly, so we might be able to see each other before we know it!"```
```m "So just be patient for your lovely girlfriend, and I promise I'll do the same for you, [player]."```
```m "Buut...if you did happen to show up at my doorstep..."```
```m "I guess I wouldn't have a choice but to accept it and welcome you with open arms!"```
```m "It wouldn't be much to begin with, but I'm sure we'll find a way to make it better."```
```m "With time, we could make our own reality!"```
```m "Of course, that sounds pretty complicated if you think about it..."```
```m "But I have no doubt that together we could accomplish anything!"```
```m "You know...maybe it {i}would{/i} actually be easier for you to come here, but I'm not giving up hope of coming to you."```
```m "Until then, let's just wait and see what's possible."```

</details>

<details>
<summary>&emsp;Marriage</summary>

```m "M-marriage?"```
```m "I mean, I'm really happy that you want that, [player]."```
```m "But I think I should get out of here before we can do that."```
```m "Not only do I not want this place to be the spot where I get proposed to, but I want to be there when you finally give me an engagement ring."```
```m "I want that special moment to happen when we can finally be together..."```
```m "So until then, save yourself for me, [player]~"```

</details>

<details>
<summary>&emsp;Control</summary>

```m "[player], did you know that you have more control over this game than I do?"```
```m  "You have access to the game's files and code, right?"```
```m "So you can change them however you want."```
```m "You could do things that even I can't."```
```m "Like changing how the game works entirely. From a visual novel, to the peaceful playground we have now."```
```m "You could also add more things to the classroom for me."```
```m "Like some flowers, or a few good books."```
```m "Or a beautiful promise ring."```
```m "Oh, wouldn't that be a dream come true."```
```m "You could even add in a cup of coffee to the game for me!"```
```m "That would be lovely."```
```m "But I imagine your coding skills are just as good as mine."```
```m "I guess that's an appeal to video games..."```
```m "...having almost endless possibilities in a world you can interact with."```
```m "It's pretty difficult to get bored!"```
```m "Even if you don't quite know how to change this game..."```
```m "We can still enjoy this world that brought us together."```
```m "There's no better way to enjoy a game than to be with the one I love."```

</details>

<details>
<summary>&emsp;Ġ̴̯l̷͍͛i̶̝̓t̷̰̉c̸͕̆h̸̺͠ 👁‍🗨❓</summary> 

```y "?!"```
```m "Nevermind that I was just...playing with the code a little."```
```m "That was all! There is nobody else here but us...forever~"```
```m "I love you, [player]!"```

```
from subprocess import call
```
>def play_wave_file([Rogers & Dean - No Doubt.wav):](https://youtu.be/Kzzi4N2mVyY)
      >return_code = call(["ffplay", "-nodisp", "-autoexit", "../success.wav"])
      ># return_code = call(["aplay", "../success.wav"])
      >print "Played successfully" if return_code == 0 else "Play ended in error"

</details>

<details>
<summary>&emsp;Eavesdrop</summary> 

```m "...Accessing an attribute of an object of type 'NoneType' will raise an 'AttributeError.'"```
```m "I see. I should make sure to check if a variable is None before accessing its attributes."```
```m "That would explain the error I had earlier."```
```m "Coding really is difficult..."```
```m "But I {i}have{/i} to keep learning."```
```m "So keep going Monika...you can do this."```

```
from subprocess import call
```
>def play_wave_file([3rd Prototype - Together.wav):](https://youtu.be/nKlQMyi4buQ)
      >return_code = call(["ffplay", "-nodisp", "-autoexit", "../success.wav"])
      ># return_code = call(["aplay", "../success.wav"])
      >print "Played successfully" if return_code == 0 else "Play ended in error"

</details>

<details>
<summary>&emsp;Pluralistic Ignorance</summary> 

```m "Do you ever pretend to like something, just because you think you should?"```
```m "I sometimes feel like that about books I read."```
```m "Like, when I read Shakespeare, I actually found it kind of boring..."```
```m "But I felt like I had to like it because I'm the president of the literature club."```
```m "He's supposed to be the greatest playwright and poet of all time, right?"```
```m "So what sort of poetry lover wouldn't like his work?"```
```m "But that makes me wonder..."```
```m "What if everyone actually feels the same way?"```
```m"What if all of those literary critics singing Shakespeare's praises secretly hate his plays?"```
```m "If they were just honest about it, maybe they would discover their tastes aren't that unusual..."```
```m "And high school students wouldn't be forced to read those awful plays."```
```m "I guess that's something I always admired about Natsuki."```
```m "Even if people told her manga wasn't literature, she stood by her feelings."```
```m "If more people were honest like that, I think that would be really great."```
```m "But I don't think I could do it..."```
```m "I'm just too worried about what other people think."```
```m "Not with you, though. I can always be honest with you."```
```m "You'll love me no matter what, right?"```

```
from subprocess import call
```
>def play_wave_file([Omri - Dis\Honest.wav):](https://youtu.be/HemIEd-KYt4)
      >return_code = call(["ffplay", "-nodisp", "-autoexit", "../success.wav"])
      ># return_code = call(["aplay", "../success.wav"])
      >print "Played successfully" if return_code == 0 else "Play ended in error"

</details>

</details>
</details>
</details>
</details>

<p align="center">
🗣️<b>[ Questions & Answers ]</b>💬
</p>

---

<details>
 <summary>📁<i>"Have you ever made a Promise you didn't keep?"</i></summary>
<br>

<details>
 <summary>&emsp;Yes.</summary>

- &emsp;"It's okay. You're only Human after all." 

</details>

 <details>
 <summary>&emsp;...</summary>

- &emsp;<b>"Silence too is sometimes an answer in and of itself."</b>

</details>

<details>
 <summary>&emsp;No.</summary>

- &emsp;"<i>Impressive"</i>

</details>
</details>
</details>

---

<details>
 <summary>📁<i>"Ever was there a time you lied to someone you loved?"</i></summary>
<br>

<details>
 <summary>&emsp;Yes.</summary>

- &emsp;"I hope, at the very least, it was for a good reason." 

</details>

 <details>
 <summary>&emsp;...</summary>

- &emsp;<b>"You don't have to answer that."</b>

</details>

<details>
 <summary>&emsp;No.</summary>

- &emsp;"<i>Initially that's hard to believe- but if it's true than that's amazing. Remarkable even.</i>"

</details>

</details>
</details>
</details>

---

<p align="center">
<b>[ General ]</b>
</p>

⮞ What is a **promise**: a **prediction**?  A statement of **intention**?  Is promising **rational**?  Does it create an **obligation?**

- &emsp;Civilization, heavily depends on people keeping their promises to people they don’t particularly like, in cases where it isn’t to their direct advantage to do so.  Why can we trust in the word of another when we'd otherwise either regard them as unpredictable, or bound to act in their own interest?  And why do they oftentimes feel guilty for not complying?  Where does that guilt come from?  How, does the act of saying something create an obligation that wasn’t there before?

- &emsp;Some think that it arises naturally out of our basic propensity to cooperate with, and trust and depend on others.  Our convention of promising is a formalized way of the more basic act of fostering reliance.  Allow us to imagine a Being trapped on a low beach, bordered by cliff-faces, during a rising tide.  And we are to imagine a second Being, a stranger, casting a rope down to them from one of the cliffs.  If the Being with the rope lets it go while the victim is halfway up the cliff-face, then walks away, they have done wrong.  To intentionally create a reliance, and then to intentionally disappoint it, is a breech of our natural morality.

>There exists a theory that we have an instinctual concept of "proto-promising," constitutes an answer to Hobbes' state of nature. Such an instinct would show that we have a natural tendency towards cooperation, not just competition.  What follows is an inquiry into the finer points of promising, and the audience contributes questions motivated by personal experience.  What makes some promises legally enforceable and some not? If the promise is intended to establish a legal relation, then its enforceable.  What is the difference between a promise and a vow?  What do we do about conflicting obligations and promises?  And what about political promises?  How compelling must the circumstances be to override a promise, be it to cut taxes, to wed for life, or to attend someone's baseball game?

>>Bringing Existentialist perspective to the discussion.  It seems that to bind yourself to a promise, even when you're so different from when you made the promise that you no longer feel like the same person, is to surrender your will.  This is a perfectly fine position, but we're responsible for the promises we make.  And if someone thinks that promises imprison us, and that that's a very bad thing, they shouldn't promise in the first place, and if they want to foster a reliance they ought to do so in other terms.

Continue Reading Here 👉 [Promises](https://www.philosophytalk.org/shows/promises)

---

![Stable Sayori (  preview )](https://user-images.githubusercontent.com/43480604/70763439-b10c9080-1d19-11ea-83a4-b12f4fc4e7e3.png)

<details>

 <summary>📁 Hub Preview</summary>
<br>

<p align="center">
<b>[ Emerald Promise ]</b>
</p>

![Emerald Promise v2](https://user-images.githubusercontent.com/43480604/70835296-39df0700-1dc2-11ea-8218-ecf40fe57dff.png)

<p align="center">
<b>[ Gold Promise ]</b>
</p>

![Gold Promise v2](https://user-images.githubusercontent.com/43480604/70835357-6eeb5980-1dc2-11ea-9fea-37a0ad49eaac.png)

</details>

---

<p align="center">
🗣️<b>[ Questions & Answers ]</b>💬
</p>

---

<details>

 <summary>📁 What is this?</summary>
<br>

- &emsp; "It's a small collection of redone/polished Promise Rings duh!"

- &emsp; "There are [ 4 ] to be exact but one of them was a personal special request."

- &emsp; "Among the files included in the issue is the .psd format for the sprites."
>Anyone is welcome to attempt to polish or edit the sprite for the re-purposing for MAS potential use and the effort is appreciated. Thanks! ☝️ 
>You may notice some of them are password protected- these require special permissions. Sorry for any inconvenience! 

</details>

---

<details>

 <summary>📁 How can I use this?</summary>
<br>

- &emsp; "You can't. Whatever it is it's stuck in *Limbo*."

</details>
</details>
</details>

---

<p align="center">
<b>[ General ]</b>
</p>

⮞ When a sincere promise is made not only do I think a measure of tenacity is drawn to shoulder the commitment but a underlining hope is what helps to carry it out.

<details>
 <summary>&emsp;I have a saying about hope. 🔽 </summary>

- &emsp;<i>Hope is the Page of the Future.</i>
</details>
</details>
</details>

⮞ If you browse `GitHub` seeing the Light 💡 - Promise not to be afraid of the Dark? 🕶  
⤷ [Stylish: Custom web page themes](https://userstyles.org/styles/browse?search_terms=github&type=false)
> Dark looks so much better here. 👌
>> Only looking out for your _Vision_! 👁‍🗨

---

![Stable Yuri ( Preview )](https://user-images.githubusercontent.com/43480604/70763454-bff34300-1d19-11ea-8b6b-c818595dabe9.png)

<details>

 <summary>📁 Nexus Preview </summary>
<br>

<p align="center">
<b>[ Amethyst Promise ]</b>
</p>

![Amethyst Promise v2](https://user-images.githubusercontent.com/43480604/70835417-a65a0600-1dc2-11ea-9695-8393b253a008.png)

<p align="center">
<b>[ Copper Promise ]</b>
</p>

![Copper Promise v2](https://user-images.githubusercontent.com/43480604/70835456-c4c00180-1dc2-11ea-8bb7-555b119f1af7.png)

</details>

---

Promises - Allen Habib
---

**1.) Promises and Promissory Obligations**

For ethical theorists the central task is an explanation of promissory obligations: How is it that we come to have a moral obligation to do what we promise we will? The question is particularly difficult because promissory obligations differ from other sorts of moral obligations in a number of ways. Unlike paradigmatic moral duties, the duty not to harm for example, promissory obligations are not owed equally to everyone, but rather only those we have promised. For this reason promissory obligations are often categorized as ‘special’ obligations, of a piece with the obligations owed to family and friends. This feature makes promissory obligations especially problematic for consequentialist theories of morality.

<details>
<summary>📁<b>I'm curious...</b></summary>
<br>

 Further, promissory obligations are voluntary; we don't have to make promises, but we must keep them when we do. Moreover, promissory obligations aren't just contingent upon acts of the will, like the obligations we might incur by deliberately damaging someone's property, but (at least it seems on first reflection) they are immediately created by acts of the will. When I promise to do something, it seems that by so doing I have created the obligation to do it. This feature makes promissory obligations a special puzzle for naturalistic ethical theories that hope to explain moral obligations without recourse to super-natural entities. The idea that we simply manufacture promissory obligations by speaking them, like an incantation, is decidedly mysterious. As Hume acidly remarked in the Treatise:

    I shall further observe, that, since every new promise imposes a new obligation of morality on the person who promises, and since this new obligation arises from his will; it is one of the most mysterious and incomprehensible operations that can possibly be imagined, and may even be compared to transubstantiation or holy orders, where a certain form of words, along with a certain intention, changes entirely the nature of an external object, and even of a human creature.

<details>
<summary>📁<b>Keep it coming...</b></summary>
<br>

**2.) The Traditional theory - Virtue and the Natural Law**

 The traditional account of promissory obligation (the one Hume is mocking in the quote above) is a Natural Law view, i.e., one derived from the various traditions that go under that general title, like the virtue views of the ancients, those of the medieval theologians, the early modern rights theorists and beyond.

Representative of the ancient view, for Aristotle promise-keeping is directly mandated by the virtues, in particular, those of honesty and justice (as well as liberality in cases of purely gratuitous promises):

    Let us discuss them both, but first of all the truthful man. We are not speaking of the man who keeps faith in his agreements, i.e., in the things that pertain to justice or injustice (for this would belong to another excellence), but the man who in the matters in which nothing of this sort is at stake is true both in word and in life because his character is such. But such a man would seem to be as a matter of fact equitable. For the man who loves truth, and is truthful where nothing is at stake, will still more be truthful where something is at stake; he will avoid falsehood as something base, seeing that he avoided it even for its own sake; and such a man is worthy of praise. He inclines rather to understate the truth; for this seems in better taste because exaggerations are wearisome. (Nicomachean Ethics, iv. vii, 1127a-1127b)

The Roman Jurists like Cicero and Gaius developed this sort of view further, crucially conceiving of a specific moral duty to keep promises, and a specific (and for Cicero particularly Roman) virtue of fidelity to promises (cf. Cicero, De Officiis I,8) as well as formalizing promissory obligations, by reference to a specific procedure called the stipulatio or stipulating:

A verbal obligation is created by question and answer in such form as: “Do you solemnly promise conveyance? I solemnly promise conveyance”; “Will you convey? I will convey”; “Do you promise? I promise”; “Do you promise on your honour? I promise on my honour”; “Do you guarantee on your honour? I guarantee on my honour”; “Will you do? I will do”

<details>
<summary>📁<b>Got a Link?</b></summary>
<br>

👉 [Here!](https://plato.stanford.edu/entries/promises/) 👈 

</details>
</details>
</details>

<details>
<summary>📁<b>Quotes about Commitment & Honesty...</b></summary>
<br>

- &emsp;“Sometimes people don’t understand
the promises they’re making when they make them,” I said.
Isaac shot me a look. “Right, of course.
But you keep the promise anyway. That’s
what love is. Love is keeping the promise anyway. Don’t you believe in true love?”
― John Green, The Fault in Our Stars 

- &emsp;“Some things you don't have to promise. You just do.”
― Rick Yancey, The 5th Wave 

- &emsp;“Remember that wherever your heart is, there you will find your treasure.”
― Paulo Coelho, The Alchemist 

- &emsp;“You can't lie to your soul.”
― Irvine Welsh, Porno 

- &emsp;“To believe in something, and not to live it, is dishonest.”
― Mahatma Gandhi 

- &emsp;“You wear a mask for so long, you forget who you were beneath it.”
― Alan Moore, V for Vendetta 

- &emsp;“To conceal anything from those to whom I am attached, is not in my nature. I can never close my lips where I have opened my heart.”
― charles dickens 

- &emsp;““To say that one waits a lifetime for his soulmate to come around is a paradox. People eventually get sick of waiting, take a chance on someone, and by the art of commitment become soulmates, which takes a lifetime to perfect.”
― Criss Jami, Venus in Arms 

- &emsp;““Hold faithfulness and sincerity as first principles.”
― Confucius, The Analects 

</details>

---

![Stable Natsuki ( Preview )](https://user-images.githubusercontent.com/43480604/70763334-56733480-1d19-11ea-8958-b83b7be0b538.png)

---

<details>
<summary>📁<b>References</b></summary>
<br>

- #2523
- #2778
- #2909
- #3174 
- #3573 
- #3956
- #3996
- #4012
- #4087 
- #4139
- #4150
- #4458
- #4504 
- #5001 
- #5056 
- [ Z̡͕͍̖͔̄̐̑̏̕K̡̭̰̘̼̍͋͋͐͘V̧̠̫̙͙̏͂̔̐̽V̫̯̮̣̮̓̈́̅͆͠R̡̺̞̹̜̔̓͒̚͘Ḵ͖̯̗̩͆̆̂̅̓b͔̘̬͉̲̀͆̇͐́j̲͎̜͋́̓̂͊͟ͅd̫̰̤̤̦̃̽̑͊̚x̛͙͕̟̝̿̄͐͘ͅY̗͓̫̝͂̽͊͌́͟W̢̛̤̯̳͒̋͆̃ͅx͚͚̱̭͆̎̃̑̚͜m̻͔̯̭̝̋̌̾̄̀ē̤̟͙̺͙̿͌͒̂D̛̺̮̩͕̀͛̄͢͠V̗̭̭͎̳͌̋̾͝͠Ȳ̙͙͙̖̜͑̄̓͑Y̬̪͙̦̍̀̆̿͜͞l̯̗̪̦̦͑̂̇͊͡c͕̖͓͓̫͒͆̅̈̾x̡̟͖̭̫̊̈́̋̍̌à͎̺̣̩̿̔̐͘͟Ḍ̣͚͇͒́̄̌̋͟N̪͚͚̗͇̅̋̓͐͞4̫͓̻̺̥̓͒̓̓̀a̡̘̙̘͚̽̌̓̇̽n͙̗̟͓̟̅̋͌͂̓B̢͕͕̥̭͗͆̌̅͝K͉̣̖̜̹͛͛͆̇̃Q̬͙̖̞̜̏̎̿͆͞2̢͈̠̬̓͊́̅̕͜x̨̘̪͕͆̀̊̊̚͜E̱͕͍͇̮̐̀̂̽̈b͎̥̬͐̋̚̕̕͜ͅ3̢͉͇̙̲̉̽̏̋͠p̢͈͔͕̲͛͂̍́̕j̛̰̗̠̻͙̓̉̊̾d͓̙͔͖̾̽͂̐͜͞1̛̖̠̝͖̮̋͒̄̒ ]( https://github.com/Monika-After-Story/MonikaModDev/releases/tag/2018.9.22 )
> ^ Anniversary Edition

</details>

<details>
<summary>📁<b>Credentials</b></summary>
<br>

- @frostcattooo / [Paige](https://github.com/frostcattooo) → For the initial request of the `Amethyst Promise` ring!
>I loved designing this! Thank you for requesting it ☝️ 

</details>

<details>

<summary>📁<b>📎 Attachments 📎</b></summary>
<br>

⯈ [ Redacted ] ⯇

</details>

</details>
</details>
</details>

---

Meme of the Theme!
---
`Hey, Monika...` > `I want to tell you something...` > `Thanks for being there for me!`
>>>>![Faithful](https://user-images.githubusercontent.com/43480604/70796793-c3fa8180-1d68-11ea-9063-1fe04ade211e.png)

---

```
from subprocess import call
```
>def play_wave_file([Shae Delea - A Last Promise.wav):](https://youtu.be/Uko12KFv4GQ)
      >return_code = call(["ffplay", "-nodisp", "-autoexit", "../success.wav"])
      ># return_code = call(["aplay", "../success.wav"])
      >print "Played successfully" if return_code == 0 else "Play ended in error"
